tests/Directory.Build.targets

```diff
@@ -2,34 +2,7 @@
 <Project>
 
   <Import Project="..\Directory.Build.targets" />
-
   <Import Project="..\eng\imports\UnitTests.targets" Condition="'$(IsUnitTestProject)' == 'true'" />
   <Import Project="..\eng\imports\IntegrationTests.targets" Condition="'$(IsIntegrationTestProject)' == 'true'" />
 
-  <PropertyGroup>
-    <_CoverageReport>$(ArtifactsTestResultsDir)$(AssemblyName).dll.coverage</_CoverageReport>
-  </PropertyGroup>
-
-  <!-- NOTICE: This relies on the unit test projects producing .coverage files. Currently, unit test projects do not produce .coverage files. -->
-  <Target Name="UploadCodecov" AfterTargets="Test" Condition="'$(UsingCodeCoverage)' == 'true' AND '$(Configuration)' == 'Debug' AND '$(IsUnitTestProject)' == 'true' AND Exists('$(_CoverageReport)')">
```

Comment:
As the comment said for this target, this target was running in our unit test pipeline, but it wasn't publishing any code coverage information.